source/analysis/AM3DBinningAB.cpp

```diff
@@ -472,8 +472,8 @@ void AM3DBinningAB::setSumRangeMin(int sumRangeMin)
     cacheUpdateRequired_ = true;
     dirtyIndices_.clear();
     reviewState();
-    emitValuesChanged();
```

Comment:
I don't think so. The problem was the analysis block being valid when the min value was larger than the max value. I have found that for analysis blocks that the signals should be emitted at the end.

Sent from my iPhone

On Nov 14, 2015, at 6:15 AM, helfrij <notifications@github.com<mailto:notifications@github.com>> wrote:


In source/analysis/AM3DBinningAB.cpp<https://github.com/acquaman/acquaman/pull/1713#discussion_r44858933>:

> @@ -472,8 +472,8 @@ void AM3DBinningAB::setSumRangeMin(int sumRangeMin)
>      cacheUpdateRequired_ = true;
>      dirtyIndices_.clear();
>      reviewState();
> -    emitValuesChanged();


Was the problem here that emitValuesChanged() was called before setModified(true)?

-
Reply to this email directly or view it on GitHub<https://github.com/acquaman/acquaman/pull/1713/files#r44858933>.